src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -523,4 +628,27 @@ public static boolean canPlayerUseItem(@Nonnull Player p, @Nullable ItemStack it
     public static @Nullable Item spawnItem(Location loc, ItemStack item, ItemSpawnReason reason) {
         return spawnItem(loc, item, reason, false);
     }
+
+    /**
+     * Helper method to check if an Inventory is empty (has no items in "storage").
```

Comment:
```suggestion
     * Helper method to check if an {@likk Inventory} is empty (has no items in "storage").
```

Reply:
While this is only docs, I really don't want to fall out of scope, that method is in the diff but has not been changed as part of this PR, likely just got jumbled at some point.

Also, {@likk} lol